test/pathToParams.test.js

```diff
@@ -0,0 +1,81 @@
+const pathToParams = require("../util/pathToParams.js");
+
+describe("pathToParams", () => {
```

Comment:
Maybe add a test case to verify allowed extensions?

Reply:
Check. Will do!